### ISSUE TITLE
amp-iframe does not show inside amp-user-notification in Chrome on Android

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amphtml#18995]
Steps to reproduce: visit https://amp-instant-app.glitch.me/ in Chrome on Android. The iframe at the bottom won't load (only an empty green banner is displayed). It might show the first time the page is loaded, but won't after reloading the page.

It works in mobile emulation in Chrome on desktop and iOS as well. 

//cc @aghassemi 

![screenshot_20181026-153213](https://user-images.githubusercontent.com/380472/47583350-74d07200-d957-11e8-9b96-55a88ad04846.png)

**Comment by aghassemi** [comment]
can repro. likely just missing a `mutateElement` call.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by sebastianbenz** [comment]
Friendly ping. This is also needed in other cases, see this [thread on SO](https://stackoverflow.com/questions/53433903/implementing-cookies-on-a-dismissible-slide-in-panel-with-amp).

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by devignerforhire** [comment]
Any news on this one? It's preventing us from consistently loading our newsletter signup form embed via an iframe in amp-user-notification on mobile.

**Comment by zhouyx** [comment]
I may know the cause to that. I've fixed a very similar issue for amp-consent. amp-user-notification in this case should handle layout for the child iframe, and the race condition here is most like amp-user-notification and AMP runtime both fail to schedule layout for the amp-iframe.

**Comment by devignerforhire** [comment]
@zhouyx Strangely, I just checked the example Sebastian posted and I can see the contents of the iframe on Chrome Android. Also my newsletter subscribe form is now showing up on chrome / android after adding the iframe placeholder. Do you know if this has this been fixed?

**Comment by zhouyx** [comment]
I don't believe we've made recent changes to amp-user-notification. Just checked @sebastianbenz example page, and seems like they changed it to using amp-img instead of amp-iframe.

**Comment by devignerforhire** [comment]
@sebastianbenz, I re-tested your example on chrome / android and the iframe doesn't show up. The odd thing is, since I added the placeholder image, my newsletter signup iframe is showing up on mobile. If I add another newsletter iframe embed for a smaller version of the signup form, I get an error saying I should use amp-fixed-ad and amp-ad instead.

Here is the desktop iframe source:
https://winefolly.com/wp-content/themes/winefolly/templates/fragments/newsletter-embed.php

and the mobile version:
https://winefolly.com/wp-content/themes/winefolly/templates/fragments/newsletter-embed-mobile.php

Or you can see a real example of our staging site:
https://winefolly.staging.wpengine.com/

For now I'll probably just use the desktop version on desktop and mobile, but it would be nice to get it working at some point.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by ampprojectbot** [comment]
This is a high priority issue but it hasn't been updated in awhile. @aghassemi Do you have any updates?

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by sebastianbenz** [comment]
Another use case via the AMP Slack:

<img width="761" alt="Screen Shot 2019-05-13 at 11 42 03" src="https://user-images.githubusercontent.com/380472/57611939-2fcb0080-7574-11e9-808a-c66d9f35aabe.png">

**Comment by wassgha** [comment]
Can't reproduce, can anyone confirm they can still see this behavior?

**Comment by sebastianbenz** [comment]
Yay! I can't either. Thanks for trying.
